--- a/c_src/xml.c
+++ b/c_src/xml.c
@@ -75,7 +75,7 @@
 inline void resize_buf(ErlNifEnv* env, struct buf *rbuf, int len_to_add)
 {
   int new_len = rbuf->len + len_to_add;
-  
+
   if (new_len > rbuf->limit) {
      while (new_len > rbuf->limit)
 	rbuf->limit *= 2;
@@ -182,7 +182,7 @@
   ERL_NIF_TERM head, tail;
   const ERL_NIF_TERM *tuple;
   int arity, ret = 1;
-  
+
   while (enif_get_list_cell(env, attrs, &head, &tail)) {
     if (enif_get_tuple(env, head, &arity, &tuple)) {
       if (arity == 2) {
@@ -207,7 +207,7 @@
       break;
     };
   };
-  
+
   return ret;
 }
 
@@ -249,7 +249,7 @@
       };
     };
   };
-  
+
   return ret;
 }
 
@@ -280,7 +280,7 @@
     };
     destroy_buf(env, rbuf);
   };
-  
+
   return enif_make_badarg(env);
 }
 
@@ -295,4 +295,3 @@
     {"element_to_binary", 1, element_to_binary}
   };
 
-ERL_NIF_INIT(xml, nif_funcs, load, NULL, NULL, NULL)
